CorsixTH/Lua/app.lua

```diff
@@ -1837,6 +1838,7 @@ function App:afterLoad()
   self.map:afterLoad(old, new)
   self.ui:afterLoad(old, new)
   self.world:afterLoad(old, new)
+  self:setCaptureMouse()
```

Comment:
Same as the world issue. I'm pretty sure `GameUI` should be capable of this lifting as a single point of control.